pages/tidal tools/vsphere.md

```diff
@@ -6,20 +6,58 @@ sidebar: main_sidebar
 permalink: vsphere.html
```

Comment:
I couldn't add the suggestion on Line#4 but can we also change the last_updated month?